applications/DAOsign.md

```diff
@@ -0,0 +1,186 @@
+|    0e. | Article                   | We will publish an **article** that explains what was done as part of the grant. And we will publish a series of articles that explains how DAOsign works from a high-level perspective. The content of the articles will be consistent with the functions at this stage. |
+|     1. | Smart Contracts           | DAOsign Smart Contract repo that includes following components: Agreement Contract, EIP712/EIP2771 standard implementation on Ink!. These smart contracts can be deployed to any substrate chain with contracts-pallet.                                                   |                                                                                                                                                                                                                                           |
+|     2. | Gas Relayer               | Off-chain component for transaction relaying                                                                                                                                                                                                                              |
+
+
+### Milestone 2 - DAOsign Application integration
+
+* **Estimated Duration:** 2 weeks
+* **FTE:**  2
+* **Costs:** 7,800 USD
+
+
+| Number | Deliverable                     | Specification                                                                                                                                                     |
+|-------:|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                         | Apache 2.0                                                                                                                                                        |
+|    0b. | Documentation                   | We will provide both inline documentation of the code and a basic tutorial that explains how a user can                                                           |
+|    0c. | Testing Guide                   | Core functions will be fully covered by unit tests to ensure functionality and robustness.                                                                        |
+|     1. | DAOsign Ink! JS SDK             | We will publish a npm/yarn package with the logic how to interact with Smart Contract from JS                                                                     |
+|     2. | DAOsign Application Integration | Integrate DAOsign application using SDK. DAOsign application will be open sourced as well. DAOsign is written using ReactJS on frontend and Typescript on backend |
```

Comment:
```suggestion
|     2. | DAOsign Application Integration | Integrate DAOsign application using SDK. DAOsign is written using ReactJS on frontend and Typescript on backend |
```

Note: It's not necessary to mention that you open source it, since it's supposed to have the same license as _0a. License_ which implies that it's open source anyway.

Reply:
Here we're referring to DAOsign core application. It is not in the scope of this grant and we explicitly pointing it will be open sourced too.

Reply:
Oh I see. I updated my suggestion; I'd still suggest you to integrate this information into the proposal.

Reply:
Done